frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx

```diff
@@ -40,7 +41,7 @@ export default class SettingsUpdatesForm extends Component {
               {jt`Metabase ${formatVersion(latestVersion)} is available.`}{" "}
               {jt`You're running ${formatVersion(currentVersion)}`}
             </span>
-            <a
+            <ExternalLink
               data-metabase-event={
```

Comment:
I think you might actually need to pass this data attribute down to the `ExternalLink` component since I don't think it handles it. This might be helpful `https://stackoverflow.com/questions/27285856/how-do-i-dynamically-set-html5-data-attributes-using-react`

Reply:
Thanks for the link.
[Added new prop](https://github.com/metabase/metabase/pull/15110/commits/a037d4f77e7faf6a06ed49ac46a93efe667beb3b), but I found a few instances of some other components in our code base that **don't** explicitly list this as a prop (`<Button />` being one of them). So, maybe it's not needed after all?

Reply:
probably an error -- data attributes will need to get added to a real dom element and react has no way of doing that on its own

Reply:
Ok, then for the future reference, those components are:
- `<Button />` (example in `frontend/src/metabase/collections/components/BulkActions.jsx`)
- `<Link />` from `react-router`
- `<RefreshWidget />` (example in `frontend/src/metabase/dashboard/components/DashboardActions.jsx`)
- `<Icon />` (example in `frontend/src/metabase/dashboard/components/DashboardHeader.jsx`)
- `<NewQueryOption />` (example in `frontend/src/metabase/new_query/containers/NewQueryOptions.jsx`)
- `<FilterPill />` (example in `frontend/src/metabase/query_builder/components/view/QuestionFilters.jsx`)
- `<QuestionDataSource />` (example in `frontend/src/metabase/query_builder/components/view/ViewHeader.jsx`)
- `<QuestionFilterWidget />` (example in `frontend/src/metabase/query_builder/components/view/ViewHeader.jsx`)
- `<QuestionSummarizeWidget />` (example in `frontend/src/metabase/query_builder/components/view/ViewHeader.jsx`)
- `<QuestionNotebookButton />` (example in `frontend/src/metabase/query_builder/components/view/ViewHeader.jsx`)
- `<NativeQueryButton />` (example in `frontend/src/metabase/query_builder/components/view/ViewHeader.jsx`)
- `<Badge />` (example in `frontend/src/metabase/questions/components/CollectionBadge.jsx`)


Searching through the code base, I realized there was already one `<ExternalLink />` instance that was using `data-metabase-event` from way before. I'll update it before this merge!
    - example: `frontend/src/metabase/query_builder/components/template_tags/TagEditorHelp.jsx`

Reply:
@daltojohnso fyi, one new commit - we both missed this one because it wasn't related to the error:
https://github.com/metabase/metabase/pull/15110/commits/4483601945b6f0b46bb5c151e47ea100e54310d1